EnglishTranslation-master/ITEM.tsv

```diff
@@ -3367,7 +3367,7 @@ ITEM_20150317_003366	Dew Recepticle
 ITEM_20150317_003367	A vessel to hold water enchanted with spiritual power.
 ITEM_20150317_003368	Flimsy Charm
 ITEM_20150317_003369	Altar Base Material.
-ITEM_20150317_003370	An enemy that dies with this charm attached to it is supposed to be under our control but.. It seems kind of shabby.
+ITEM_20150317_003370	An enemy that dies with this charm attached to it is supposed to be under our control but... It seems kind of shabby.
```

Comment:
What about

`Any enemy that dies with this charm attached to it, is supposed to be under our control... It seems kind of shabby.`
?